src/lib/autoLauncher.ts

```diff
@@ -38,24 +39,30 @@ Comment=${appName} startup script
 Exec=${appPath}${hiddenArg}
 Icon=${appName}
   `
-    await fs.writeFile({ contents, path: autostartAppFile }).catch(console.error)
+    await fs.writeFile({ contents, path: autostartAppFile }).catch((error) => {
+      errorLogger(error)
+    })
     response.stdout.push("success")
     return response
   },
   async disable(appName: string): Promise<ChildReturnData> {
     const autostartAppFile = await this.getAutostartFilePath(appName)
     const response: ChildReturnData = { stderr: [], stdout: [] }
-    await fs.removeFile(autostartAppFile).catch(console.error)
+    await fs.removeFile(autostartAppFile).catch((error) => {
+      errorLogger(error)
+    })
     response.stdout.push("success")
     return response
   },
   async isEnabled(appName: string): Promise<boolean> {
     try {
       const autostartAppFile = await this.getAutostartFilePath(appName)
-      await fs.readTextFile(autostartAppFile).catch(console.error)
+      await fs.readTextFile(autostartAppFile).catch((error) => {
+        errorLogger(error)
+      })
       return true
     } catch (error) {
-      console.error(error)
+      errorLogger(error as string)
```

Comment:
error can be both `string` and instance of `Error`, so should be something like:
```
if (typeof error === "string") {
        errorLogger(error)
    } else if (error instanceof Error) {
        errorLogger(error.message)
    }
    ```

Reply:
same in other places when you catch errors

Reply:
oh ok thanks!